### ISSUE TITLE
Unity Package for iOS 2.10.0 in scoped registry throws error in Unity console

### ISSUE THREAD

**Issue by RuiBEAT** [Azure/azure-spatial-anchors-samples#274]
I have just updated Azure Spatial Anchors SDK (Core, for Android and for iOS) to version 2.10.0 from 2.9.0 using the Package Manager, with the Scoped Registry configured as followed:

```
{
      "name": "Azure Mixed Reality Services",
      "url": "https://pkgs.dev.azure.com/aipmr/MixedReality-Unity-Packages/_packaging/Unity-packages/npm/registry/",
      "scopes": [
        "com.microsoft.azure.spatial-anchors-sdk"
      ]
}
```

After the update, the following errors are persistently displayed on the console every time Unity gets focus:

> Asset Packages/com.microsoft.azure.spatial-anchors-sdk.ios/Plugins/iOS/AzureSpatialAnchors.framework/_CodeSignature/CodeResources has no meta file, but it's in an immutable folder. The asset will be ignored.
> 
> Asset Packages/com.microsoft.azure.spatial-anchors-sdk.ios/Plugins/iOS/AzureSpatialAnchors.framework/_CodeSignature/CodeResources has no meta file, but it's in an immutable folder. The asset will be ignored.

It's remarkable to say that right now I am working for the Android platform, and that's how Unity is configured. Trying to generate version (Android) doesn't work either. Doing so, it displays the following errors per console.

> Build completed with a result of 'Failed' in 11 seconds (11394 ms)
> UnityEngine.GUIUtility:ProcessEvent (int,intptr,bool&) (at /Users/bokken/buildslave/unity/build/Modules/IMGUI/GUIUtility.cs:189)
> 
> UnityEditor.BuildPlayerWindow+BuildMethodException: 4 errors
>   at UnityEditor.BuildPlayerWindow+DefaultBuildMethods.BuildPlayer (UnityEditor.BuildPlayerOptions options) [0x002be] in /Users/bokken/buildslave/unity/build/Editor/Mono/BuildPlayerWindowBuildMethods.cs:190 
>   at UnityEditor.BuildPlayerWindow.CallBuildMethods (System.Boolean askForBuildLocation, UnityEditor.BuildOptions defaultBuildOptions) [0x00080] in /Users/bokken/buildslave/unity/build/Editor/Mono/BuildPlayerWindowBuildMethods.cs:95 
> UnityEngine.GUIUtility:ProcessEvent (int,intptr,bool&) (at /Users/bokken/buildslave/unity/build/Modules/IMGUI/GUIUtility.cs:189)

**Reverting back to version 2.9.0 the "for iOS" dependency in the same scoped registry makes everything work properly again (even keeping 2.10.0 version for both Core and Android).**

**Comment by MehranAzimi-msft** [comment]
Thanks for reporting this issue. We were able to repro it on our side in our samples, and are considering to release a hot fix soon to address this issue. Note in most cases in our tests the error you reported is shown in the Unity Editor, however, Unity can still build and generate the Xcode project, which then can be compiled and deployed successfully. In one machine the same Unity Editor version and sample project could not create the Xcode project, so it seems there is a Unity editor setting that makes a difference. 

Can you please try importing the UPM packages not by using ScopedRegistry, but by downloading the UPM package from [here](https://dev.azure.com/aipmr/MixedReality-Unity-Packages/_packaging?_a=package&feed=Unity-packages&package=com.microsoft.azure.spatial-anchors-sdk.ios&protocolType=Npm&version=2.10.0)  and importing the tarball to your project by following these [instructions](https://docs.unity3d.com/Manual/upm-ui-tarball.html). With this method, you can edit the content of the tarball and delete the Plugins\iOS\AzureSpatialAnchors.framework\_CodeSignature folder, which should unblock you from using the package. 

Note we could not repro the Android issue you are reporting in our samples, and from error it seems it is not related to ASA. Could you please try importing via tarball method mentioned above and delete the _CodeSignature folder from iOS package to see if it makes a difference with your Android issue? It also worth double checking if there is any other package version that is changed. 

Please also let us know which Unity editor version you are using.

**Comment by MehranAzimi-msft** [comment]
We just published ASA SDK 2.10.1 hotfix that contains a fix for this issue.

**Comment by RuiBEAT** [comment]
Hi MehranAzimi-msft!

The issue seems fixed in the 2.10.1. No error is thrown in the console and the Android version of the app builds flawlessly.

About the Android issue, the point is that the faulty OS package seemed to interfere somehow in the Android build process. Reverting only the iOS package to 2.9.0 (keeping 2.10.0 for the Core and Android), the Android version was built perfectly. As soon as the iOS package it was updated to 2.10.0, errors appeared on the console and, paradoxically, the Android version gave build errors. Anyway, everything seems fixed in the 2.10.1 version.

Thanks for the technical support!
